### PR TITLE
Allow user pick their own logger with custom config instead

### PR DESCRIPTION
**Pull request by trinhnx**
An implementation for this discussion https://github.com/line/line-bot-sdk-java/issues/1415

### CODE REVIEW

File: clients/line-bot-client-base/src/main/java/com/linecorp/bot/client/base/ApiClientBuilder.java [hunk 1]
```diff
@@ -97,6 +97,8 @@ public ApiClientBuilder(URI apiEndPoint, Class<T> clientClass, ExceptionBuilder
 
     private Integer maxRequestsPerHost = 5;
 
+    private boolean usingDefaultLogger = true; // Just kept the same behavior as the original code
```
Comment (mokuzon) [inline clients/line-bot-client-base/src/main/java/com/linecorp/bot/client/base/ApiClientBuilder.java:100]: This comment is redundant, people who do not see this PR will be confused by the term original code and should be deleted.

File: clients/line-bot-client-base/src/main/java/com/linecorp/bot/client/base/http/HttpInterceptor.java [hunk 2]
```diff
@@ -18,6 +18,18 @@
...
+import okhttp3.Interceptor;
+import okhttp3.logging.HttpLoggingInterceptor;
+
 public interface HttpInterceptor {
     HttpResponse intercept(HttpChain httpChain) throws IOException;
+
+    static Interceptor loggingInterceptor() {
+        final Logger slf4jLogger = LoggerFactory.getLogger("com.linecorp.bot.client.wire");
+        return new HttpLoggingInterceptor(slf4jLogger::info)
+                .setLevel(HttpLoggingInterceptor.Level.BODY);
```
Comment (mokuzon) [inline clients/line-bot-client-base/src/main/java/com/linecorp/bot/client/base/http/HttpInterceptor.java:33]: [Q] Why is code similar to ApiClientBuilder's buildLoggingInterceptor() needed here?

**Review by mokuzon — COMMENTED** [review]
Thanks for contributing!!! & sorry for the very late review.

File: clients/line-bot-client-base/src/main/java/com/linecorp/bot/client/base/http/HttpInterceptor.java (hunk 2 above)
Comment (trinhnx) [reply clients/line-bot-client-base/src/main/java/com/linecorp/bot/client/base/http/HttpInterceptor.java:33]: nice catch @mokuzon , it is duplicated!
I removed it

File: clients/line-bot-client-base/src/main/java/com/linecorp/bot/client/base/ApiClientBuilder.java (hunk 1 above)
Comment (trinhnx) [reply clients/line-bot-client-base/src/main/java/com/linecorp/bot/client/base/ApiClientBuilder.java]: i deleted it already

**Comment by trinhnx** [comment]
@Yang-33 san  / @mokuzon san, could we merge this PR and close this one?

**Review by mokuzon — APPROVED** [review]

**Comment by mokuzon** [comment]
@trinhnx Sorry for the delay! We have released it as 9.4.0. Thank you so much for your contribution!
https://search.maven.org/artifact/com.linecorp.bot/line-bot-messaging-api-client/9.4.0/jar

**Pull request merged**